packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.pcss

```
@@ -6,7 +6,7 @@
   @mixin set-font-size var(--atomic-text-base);
 }
 
-[part='source-title'] a {
+a[part='source-title'] {
```

```suggestion
[part='source-title'] {
```
just using the part itself should be ok. Allow users to more easily update the styling